docs/src/julia.md

```diff
@@ -0,0 +1,69 @@
+# Julia Background
```

Comment:
it's nice to add this! My only comment is that this should go at/near the end of the table of contents